.github/workflows/on_pull_request.yml

```diff
@@ -77,6 +79,7 @@ jobs:
           MATCH_KEYCHAIN_PASSWORD: ${{ secrets.MATCH_KEYCHAIN_PASSWORD }}
           PR_NUMBER: ${{ github.event.pull_request.number }}
           APPETIZE_API_TOKEN: ${{ secrets.APPETIZE_API_TOKEN }}
+          ACCESS_TOKEN: ${{ secrets.PRIMER_PODSPEC_ACCESS_TOKEN }}
```

Comment:
Can this be renamed to GITHUB_ACCESS_TOKEN (on both sides), since it can be used for other github actions as well?